index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <title>Rekupe</title>
+</head>
+<body>
+<header class="outer-container" id="outer-container-header">
+    <div class="inner-container header-content">
+        <nav>
+            <ul>
+                <li><a href="#rs" style="text-decoration: none">Case Studies</a></li>
+                <li><a href="#services" style="text-decoration: none">Services</a></li>
+                <li><a href="#contact" style="text-decoration: none">Contact Us</a></li>
+            </ul>
+        </nav>
+        <article class="header">
+            <h1 id="header-h1">Your website is an investment in the succes of your business.</h1>
+            <p id="header"> We are here to help you succeed.</p>
+            <button type="button" id="header-button"> let's work together</button>
+            <img src="../assets/squiggle_line_stripe.svg" alt="styling-golfje" id="styling-wave-header"/>
```

Comment:
Het pad naar het plaatje klopt niet
```suggestion
            <img src="./assets/squiggle_line_stripe.svg" alt="styling-golfje" id="styling-wave-header"/>
```
of
```suggestion
            <img src="assets/squiggle_line_stripe.svg" alt="styling-golfje" id="styling-wave-header"/>
```